plugins/inputs/vsphere/vsphere_test.go

```diff
@@ -230,36 +227,6 @@ func TestParseConfig(t *testing.T) {
 	require.NotNil(t, tab)
 }
 
-func TestThrottledExecutor(t *testing.T) {
```

Comment:
This test is racy and somewhat slow to run, I don't think it provides enough value to keep.